.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,14 +2,19 @@
 
 ### Screenshot / video of UI
 
-<!-- If this PR is changing UI, please include 
+<!-- If this PR is changing UI, please include
 screenshots or screencasts showing the difference -->
 
 ### What issues does this PR fix or reference?
 
-<!-- Include any related issues from Podman Desktop 
+<!-- Include any related issues from Podman Desktop
 repository (or from another issue tracker). -->
 
 ### How to test this PR?
 
 <!-- Please explain steps to reproduce -->
+
+### Do you have unit tests?
+
+<!-- In case of adding new or updating actual functionality
+you need to provide unit tests -->
```

Comment:
to me it's duplicating the previous entry line 13
if there are unit tests I usually add it there